scala/scala.bzl

```diff
@@ -174,22 +190,11 @@ rm -rf {out}_tmp_expand_srcjars
   outs = [ctx.outputs.jar]
   if buildijar:
     outs.extend([ctx.outputs.ijar])
+  ins = list(jars) + list(dep_srcjars) + list(srcjars) + list(sources) + ctx.files.srcs + ctx.files.plugins + ctx.files.resources + ctx.files._jdk + ctx.files._jar + ctx.files._scalasdk + [ctx.outputs.manifest, ctx.file._ijar, ctx.file._scalac, ctx.file._java, scalac_args_file]
```

Comment:
I wrote about it in the comments. Tried to but couldn't due to indentation.
Python noob and probably making a small mistake.

On יום ב׳, 20 ביוני 2016 at 6:46 P. Oscar Boykin <notifications@github.com>
wrote:

> In scala/scala.bzl
> <https://github.com/bazelbuild/rules_scala/pull/71#discussion_r67633561>:
>
> > @@ -174,22 +190,11 @@ rm -rf {out}_tmp_expand_srcjars
> >    outs = [ctx.outputs.jar]
> >    if buildijar:
> >      outs.extend([ctx.outputs.ijar])
> > +  ins = list(jars) + list(dep_srcjars) + list(srcjars) + list(sources) + ctx.files.srcs + ctx.files.plugins + ctx.files.resources + ctx.files._jdk + ctx.files._jar + ctx.files._scalasdk + [ctx.outputs.manifest, ctx.file._ijar, ctx.file._scalac, ctx.file._java, scalac_args_file]
>
> can we wrap this line again, it is really long.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/bazelbuild/rules_scala/pull/71/files/99bbd657a1f504b46d6f246149d8321de56abf24#r67633561>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABUIF-MAUi7kC0iyyNjyRLchNcFWDbfjks5qNg0vgaJpZM4Ir-mk>
> .
>